indico/web/forms/widgets.py

```diff
@@ -198,6 +198,42 @@ def __call__(self, field, **kwargs):
                                                      search_url=self.search_url, choices=getattr(field, 'choices', []))
 
 
+class LocationWidget(JinjaWidget):
+    """Renders a collection of fields to represent location"""
+
+    def __init__(self):
+        super(LocationWidget, self).__init__('forms/location_widget.html', single_line=True)
```

Comment:
yeap, I want the label to be vertically aligned in the middle of the input fields.
![image](https://cloud.githubusercontent.com/assets/1579899/12745405/59a9856e-c999-11e5-93be-db51d1c99460.png)